packages/components/select/src/select-button.ts

```diff
@@ -13,6 +19,9 @@ declare global {
   }
 }
 
+Icon.register(faCircleXmark, fasCircleXmark);
```

Comment:
Maybe those icons need to be added to the `core.json` file instead of registering them here?

<img width="640" alt="image" src="https://github.com/user-attachments/assets/6ed94a55-c29e-4adb-90fc-ef3c643ee986" />